src/core/drive/visit.ts

```diff
@@ -251,6 +262,15 @@ export class Visit implements FetchRequestDelegate {
     }
   }
 
+  goToSamePageAnchor() {
```

Comment:
Given that scrolling is now handled by `render`, I wonder if this should be renamed? It's still needed, but it just bypasses page rendering.

Reply:
I think the Snapshot re-structuring will yield many opportunities to refactor this changeset. I'm mostly interested in restoring a passing test suite so that we have a safety net for those changes.